wlauto/workloads/googlephotos/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -221,27 +220,23 @@ public void closeAndReturn(final boolean dontsave) throws Exception {
                                          .className("android.widget.ImageButton"));
         navigateUpButton.waitForExists(viewTimeout);
         navigateUpButton.click();
+        sleep(1);
+        navigateUpButton.click();
     }
 
     private void gesturesTest() throws Exception {
         String testTag = "gesture";
 
         // Perform a range of swipe tests while browsing photo gallery
         LinkedHashMap<String, GestureTestParams> testParams = new LinkedHashMap<String, GestureTestParams>();
-        testParams.put("swipe_left", new GestureTestParams(GestureType.UIDEVICE_SWIPE, Direction.LEFT, 10));
```

Comment:
no point swiping when there is only 1 image. Removed, with the plan to extend this test (or make a new test) that tests swiping/scrolling with 100s of images